packages/happo-core/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "happo-core",
+  "version": "5.0.0",
+  "description": "Command-line tool to visually diff JavaScript components",
+  "repository": "https://github.com/Galooshi/happo/tree/master/packages/happo-core",
+  "author": "",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "../../node_modules/.bin/babel src -d lib --ignore __tests__",
+    "prepublish": "npm run build"
```

Comment:
i think so. i'm not really sure. lerna has some commands that let you run commands for all of the packages. `lerna run foo` would run `npm run foo` in each of the packages where a `foo` script is defined. For publishing, lerna lets you publish all of the packages at once. they have an "independent" mode where you can control the versioning of each package independently, or the default which is to have all the packages increment together (which is how babel and react work, for example)